assets/sass/components/_home-content.scss

```diff
@@ -249,6 +249,6 @@
 
 .hub-content-block__title-bar {
   display: flex;
-  align-items: flex-start;
-  justify-content: space-between;
+  align-items: center;
+  justify-content: flex-start;
 }
```

Comment:
Have these changes been checked with the current homepage? (As it looks like they might effect it).

Reply:
the hub-content-block is only used on the new homepage